src/js/chapterDOM.js

```diff
@@ -1,80 +1,96 @@
-  let section;
+  let dragContainer; 
 
   const hideStoryContent = () => {
     const story = document.querySelector("p");
-    const images = document.querySelector(".images");
+    const images = dragContainer.querySelectorAll(".picture")
     const nextButton = document.querySelector(".next");
-    const question = document.querySelector(".question");
+    const question = document.querySelector(".chapter-question");
 
+    
     story.classList.toggle("hide");
-    images.classList.toggle("hide");
+    images.forEach(img => img.classList.toggle("hide"))
     nextButton.classList.toggle("hide");
     question.classList.toggle("hide");
   };
 
   const createButtons = () => {
-    const homeButton = document.createElement("button");
-    const nextChapterButton = document.createElement("button");
-
-    homeButton.textContent = "home";
+    const homeButton = document.createElement("img");
     homeButton.classList.add("home");
+    homeButton.src = homeImage;
     //homeButton.addEventListener('click', showHomeScreen)
 
-    nextChapterButton.textContent = "Next";
+    const nextChapterButton = document.createElement("img");
     nextChapterButton.classList.add("next");
+    nextChapterButton.src = arrow;
     nextChapterButton.addEventListener("click", hideStoryContent);
-
+    
     return { homeButton, nextChapterButton };
   };
 
   const setupImages = () => {
+    dragContainer = document.createElement("div");
     const images = chapterObj.getImages();
-    const gallery = document.createElement("div");
-
-    gallery.className = "images";
 
     for (let image of images) {
       const classArr = image.cssClass.split(" ");
 
       const picture = document.createElement("img");
       picture.src = image.url;
-      picture.classList.add(classArr[0]);
-      picture.classList.add(classArr[1]);
+
+      const pictureContainer = document.createElement("div");
+      pictureContainer.classList.add("picture")
+      pictureContainer.classList.add(classArr[0]);
+      pictureContainer.classList.add(classArr[1]);
+      pictureContainer.append(picture);
 
       if (picture.className !== "drop-container") {
-        makeDraggable(picture, section);
+        makeDraggable(pictureContainer, dragContainer);
       }
-
-      gallery.append(picture);
+      
+      dragContainer.append(pictureContainer);
     }
 
     makeDroppable(
-      gallery.querySelector(".right"),
-      gallery.querySelector(".drop-container"),
+      dragContainer.querySelector(".right"),
+      dragContainer.querySelector(".drop-container"),
       () => callback(chapterObj)
     );
 
-    return gallery;
+    return dragContainer;
   };
 
   const setupChapterPage = () => {
-    section = document.createElement("section");
     const { homeButton, nextChapterButton } = createButtons();
+    
+    const title = document.createElement("H1");
+    title.classList.add("chapter-title");
+    title.textContent = "Chapter 1";
```

Comment:
We can dynamically set chapter number by using getChapterNumber method:
`title.textConent = 'Chapter ${chapterObj.getChapterNumber()}'`

Reply:
Thx! I didn't know that it would be so easy. =)